js/containers/report/iframe-standalone-app.js

```diff
@@ -0,0 +1,104 @@
+import React, { PureComponent } from "react";
+import { connect } from "react-redux";
+import {
+  fetchAndObserveData, hideCompareView,
+  hideUnselectedQuestions, showUnselectedQuestions, setNowShowing,
+  setAnonymous, trackEvent } from "../../actions/index";
+import DataFetchError from "../../components/report/data-fetch-error";
+import LoadingIcon from "../../components/report/loading-icon";
+import InteractiveIframe from "../../components/report/interactive-iframe";
+
+import "../../../css/report/report-app.less";
+import "../../../css/report/iframe-standalone-app.less";
+import config from "../../config";
+
+class IframeStandaloneApp extends PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    const { fetchAndObserveData } = this.props;
+    fetchAndObserveData();
+  }
+
+  renderIframe() {
+    const { report, iframeQuestionId, answers } = this.props;
+
+    const question = report.get("questions").get(iframeQuestionId);
+
+    const studentId = config("studentId");
+    const answer = answers.filter(a =>
+      a.get("questionId") === iframeQuestionId &&
+      a.get("platformUserId") === studentId
+    ).first();
+
+    if (!answer) {
+      const errorText =
+        !iframeQuestionId ? "Parameter 'iframeQuestionId' is missing" :
+        !studentId ? "Parameter 'studentId' is missing" :
+        `No data for question '${iframeQuestionId}' by student '${studentId}'`;
+      return <DataFetchError error={{title: "Unable to fetch data", body: errorText}} />;
+    }
+
+    let url;
+    let state;
+    // There are two supported answer types handled by iframe question: simple link or interactive state.
+    if (answer.get("type") === "external_link") {
+      // Answer field is just the reportable URL. We don't need any state.
+      url = answer.get("answer");
+      state = null;
+    } else if (answer.get("type") === "interactive_state") {
+      // URL field is provided by question. Answer field is a state that will be passed
+      // to the iframe using iframe-phone.
+      url = question.get("url");
+      state = answer.get("answer");
+    }
+    return (
+      <InteractiveIframe src={url} state={state} style={{border: "none"}} width="100%" height="100%" />
+    );
```

Comment:
With the exception of the width and height this is identical to: 
https://github.com/concord-consortium/portal-report/blob/172324236-standalone-iframe/js/components/report/iframe-answer.js#L78-L95